tests-invoke

```diff
@@ -96,7 +96,9 @@ def main():
                 time.sleep(int(os.getenv("TEST_INVOKE_SLEEP", "60")))
         return_code = p.returncode
         sys.stderr.write(f"Test run finished, return code: {return_code}\n")
-        if not api.has_open_prs(opts.revision) and not collision_detected and return_code != 0:
+        # --pull-number is absent for nightly runs; but we special-case "pull-0",
+        # so if someone passes "0" that also doesn't count
+        if not opts.pull_number and not collision_detected and return_code != 0:
```

Comment:
Then we should check for `not opts.pull_number` and `opts.pull_number != 0`

Reply:
I'm confused. We want to run the "then" part for nightly runs without a PR, but never for PRs. So if `opts.pull_number` is truthy (a number > 0), `not opts.pull_number` is False, and the `if` is skipped. But if it's absent (None) or 0, then `not opts_pull_number` is truthy.

I.e. do you think it's wrong, or you just prefer it to be more explicit?

`if (opts.pull_number is None or opts.pull_number == 0) and ...`

?

Reply:
Aaah, it's time to recharge. Right, I thought we checked for `is None` originally so the check is correct.